src/components/CommandBar/CommandBarSelectionMixedInput.tsx

```diff
@@ -92,9 +110,10 @@ export default function CommandBarSelectionMixedInput({
           (!hasSubmitted || canSubmitSelection || 'text-destroy-50')
         }
       >
-        {canSubmitSelection
-          ? 'Select objects in the scene'
-          : 'Select code or objects in the scene'}
+        {canSubmitSelection &&
+        (selection.graphSelections.length || selection.otherSelections.length)
+          ? getSelectionTypeDisplayText(selection) + ' selected'
+          : 'Select code/objects, or skip'}
```

Comment:
@franknoirot I changed the copy again haha.

When I was using the app, it just struck me that it feels like a required option, and given the intention Jordan was going for, I can totally see him asking for a second take.

Happy for you to change it back again.

Reply:
Haha no this is fine. We should come back and make the `, or skip` conditional on if `isRequired === true`